supervisor/api/__init__.py

```diff
@@ -103,16 +104,36 @@ def _register_host(self) -> None:
         self.webapp.add_routes(
             [
                 web.get("/host/info", api_host.info),
-                web.get("/host/logs", api_host.logs),
+                web.get("/host/logs", api_host.advanced_logs),
```

Comment:
So with that we make the advanced logging (systemd-journal-gatewayd logging) already mandatory today. I think I'd have preferred if we would have gone a two step approach, essentially reverting to the old approach if systemd-journal-gatewayd is not there (especially since the Supervised installation only recently [got that change in](https://github.com/home-assistant/supervised-installer/pull/235/files). OS exposes `systemd-journal-gatewayd` since 7.0 so from a ADR0015 point of view we are well past that. But we still have almost 3k people on pre-7.0 OS versions :cold_sweat: ).

That said, I am ok with moving fully to it, we just should make sure the system behaves somewhat sensible in case it has the `systemd-journal-gatewayd` socket missing. Especially, how will the user realize what is wrong?

From what I can tell the API will throw `HostNotSupportedError` in that case. Is the frontend behaving somewhat sensible in this situation?